.github/dependabot.yml

```
@@ -1,6 +1,31 @@
+    schedule:
+      interval: 'weekly'
+    reviewers:
+      - 'Kravalg'
+    groups:
+      all-deps:
+        patterns:
+          - '*'
+    commit-message:
+      prefix: 'feat'
+      prefix-development: 'chore'
+      include: 'scope'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)